tests/test_sklearn_decoder.py

```diff
@@ -86,3 +101,10 @@ def test_sklearn_decoder(decoder):
         decoder.fit(X, y_str)
     with pytest.raises(ValueError, match="Invalid.*shape"):
         decoder.fit(X, y_d_short)
+
+
+def test_dtype_checker():
+    assert cebra_sklearn_decoder._is_floating(torch.Tensor([4.5]))
+    assert cebra_sklearn_decoder._is_integer(torch.LongTensor([4]))
+    assert cebra_sklearn_decoder._is_floating(np.array([4.5]))
+    assert cebra_sklearn_decoder._is_integer(np.array([4]))
```

Comment:
minor thing, can we add negative examples to this test as well? i.e. make a few tests that should raise an error?